packages/@aws-cdk/aws-backup/lib/plan.ts

```diff
@@ -0,0 +1,256 @@
+   * @see https://docs.aws.amazon.com/aws-backup/latest/devguide/creating-a-backup-plan.html
+   */
+  readonly schedule: Schedule;
+  /**
+   * The lifecycle specification for managing the backup resources.
+   * @see https://docs.aws.amazon.com/aws-backup/latest/devguide/creating-a-backup-plan.html
+   * @default resources have not lifecycle
+   */
+  readonly lifecycle?: Lifecycle;
+  /**
+   * The time in which a backup job has to be completed before being cancelled.
+   * @default backup window is set to start at 5AM UTC and lasts 8 hours.
+   */
+  readonly completionWindow?: Duration;
+  /**
+   * The time after a backup is scheduled before a job is canceled if it doesn't start successfully.
+   * @default backup window is set to start at 5AM UTC and lasts 8 hours.
+   */
+  readonly startWindowAfter?: Duration;
+}
+
+/**
+ * Represents a Backup Plan.
+ */
+export interface IBackupPlan extends IResource {
+  /**
+   * The backup vault where the plan will save the backups.
+   * @attribute
+   */
+  readonly vault: IBackupVault;
+  /**
+   * The backup plan name.
+   * @attribute
+   */
+  readonly backupPlanName: string;
+  /**
+   * The backup plan's ARN.
+   * @attribute
+   */
+  readonly backupPlanArn: string;
+  /**
+   * The backup plan's ID.
+   * @attribute
+   */
+  readonly backupPlanId: string;
+  /**
+   * A string representation of the backup plan's version ID
+   * @attribute
+   */
+  readonly backupPlanVersionId: string;
+}
+
+abstract class BackupPlanBase extends Resource implements IBackupPlan {
+  public abstract readonly vault: IBackupVault;
+  public abstract readonly backupPlanName: string;
+  public abstract readonly backupPlanArn: string;
+  public abstract readonly backupPlanId: string;
+  public abstract readonly backupPlanVersionId: string;
+}
+
+/**
+ * Backup plan attributes.
+ */
+export interface BackupPlanAttributes {
+  /**
+   * A string representation of the backup plan name
+   */
+  readonly backupPlanName: string;
+  /**
+   * A string representation of the backup plan ARN.
+   */
+  readonly backupPlanArn: string;
+  /**
+   * A string representation of the backup plan version ID.
+   */
+  readonly backupPlanVersionId: string;
+  /**
+   * A string representation of the backup plan ID.
+   */
+  readonly backupPlanId: string;
+  /**
+   * An object representation of a backup vault.
+   */
+  readonly vault: IBackupVault;
+}
+
+/**
+ * Properties of the Backup Plan.
+ */
+export interface BackupPlanProps {
+  /**
+   * The name of the backup plan
+   */
+  readonly backupPlanName: string;
+  /**
+   * A list of plan rules to assign to the plan.
+   */
+  readonly rules: PlanRule[];
+  /**
+   * The backup vault where all backup resources will be stored in.
+   */
+  readonly vault: IBackupVault;
+}
+
+/**
+ * A Backup Plan specifies where the backup resources will be stored and
+ * what rules will the plan will follow.
+ */
+export class BackupPlan extends BackupPlanBase {
```

Comment:
Here also we can have nice static methods that mimic the console experience: `public static dailyWeeklyMonthly5YearRetention(scope: Construct, id: string, backupVault: IBackupVault)` etc.